colour/models/rgb/datasets/__init__.py

```diff
@@ -18,6 +18,7 @@
 from .color_match_rgb import COLOR_MATCH_RGB_COLOURSPACE
 from .dcdm_xyz import DCDM_XYZ_COLOURSPACE
 from .dci_p3 import DCI_P3_COLOURSPACE, DCI_P3_P_COLOURSPACE
+from .display_p3 import DISPLAY_P3_COLOURSPACE, DISPLAY_P3_COLOURSPACE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'DISPLAY_P3_COLOURSPACE' from line 21](https://app.codacy.com/app/colour-science/colour/pullRequest?prid=4677381)